### PR TITLE
#1021: added skip-links

### PR DESCRIPTION
**Pull request by anu-007**
added skip-links in the web-app.

### CODE REVIEW

File: src/backend/_scss/application.scss [hunk 1]
```diff
@@ -1384,3 +1384,26 @@ a {
...
 }
+
+.skip {
+  position: absolute;
+  top: -1000px;
+  left: -1000px;
+  height: 1px;
+  width: 1px;
+  text-decoration:none;
+  color: #ffffff;
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: please use color codes mentioned in config.scss

File: src/backend/templates/event.hbs [hunk 2]
```diff
@@ -32,10 +32,11 @@
     </head>
 
     <body onload="datafetcher()">
+      <a href="#content" class="skip">Skip to content</a>
     {{> scroll}}
     <a id="top"></a>
     {{> navbar}}
-    <div class="carousel slide" style="margin-bottom: 0;">
+    <div class="carousel slide" style="margin-bottom: 0;" id="content">
```
Comment (aayusharora) [inline src/backend/templates/event.hbs]: Avoid Bad practices : id is a unique attribute and hence can not be repeated with html elements, use classes instead.

Comment (anu-007) [inline src/backend/templates/event.hbs]: @aayusharora thanks for the review should I send a new PR again with these corrections.

Comment (aayusharora) [inline src/backend/templates/event.hbs]: No, just update this PR with new changes :+1:

Comment (anu-007) [inline src/backend/templates/event.hbs]: I tried to use classes instead of id attribute but "href" in anchor tag is only working with id.

Comment (aayusharora) [inline src/backend/templates/event.hbs]: @anu-007 That make sense `<a>` element take its id and matches with all id on the page to scroll.

Comment (aayusharora) [inline src/backend/templates/event.hbs]: @anu-007 The point is ` Don't make the id same for all sections `  Like in your case you have used content in all files.

**Comment by aayusharora** [comment]
@anu-007 Squash your commits please :)

**Comment by aayusharora** [comment]
@anu-007 After your changes a blue border box appears every time the content is clicked. Please check why it is happening. Please check how this was in the  previous version. Thanks :)

File: src/backend/_scss/application.scss [hunk 3]
```diff
@@ -1384,3 +1384,50 @@ a {
...
+
+.schedule-container:focus{
+  opacity: 0.9;
+}
+
+.left-border{
+  padding-right: 0px;
+  margin-bottom: -1px;
+  padding-bottom:10px;
+  border: none;
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: @anu-007 Replace `border: none`.

File: src/backend/_scss/application.scss [hunk 4]
```diff
@@ -1384,3 +1384,50 @@ a {
...
+}
+
+.schedule-container:focus{
+  opacity: 0.9;
+}
+
+.left-border{
+  padding-right: 0px;
+  margin-bottom: -1px;
+  padding-bottom:10px;
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: @anu-007 Follow this please

File: src/backend/_scss/application.scss [hunk 5]
```diff
@@ -1384,3 +1384,50 @@ a {
...
+  background: none; 
+  outline:0;
+}
+
+.schedule-container:focus{
+  opacity: 0.9;
+}
+
+.left-border{
+  padding-right: 0px;
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: @anu-007 Remove `px` from here

**Comment by aayusharora** [comment]
@anu-007 This PR contains history from other users 
`CosmicCoder96 and others added some commits 12 days ago`
Can you please correct it.

**Comment by aayusharora** [comment]
Closing this. This contains code from other users. Please make a new PR @anu-007

**Pull request closed without merging**